node/resolvers/search/facets.ts

```diff
@@ -0,0 +1,11 @@
+export const resolvers = {
+  Facet: {
+    values: (facet: any, args: any) => {
+      const { values } = facet
+      return values.slice(args.from ?? 0, args.to)
+    },
+    quantity: (facet: any) => {
+      return facet.values.length || 0
```

Comment:
When will `facet.values.length` be something other than `0`?

```suggestion
      return facet.values.length || 0
```